backend/package.json

```diff
@@ -11,7 +11,9 @@
     "start": "UPDATE_DATA=true DEV=true ts-node ./src/app.ts",
     "clean": "rm -rf node_modules && rm -rf build",
     "test": "jest --forceExit",
-    "build": "tsc --project ."
+    "build": "tsc --project .",
+    "redis:reset_day": "ts-node ./src/redis/redisRefresh.ts day",
```

Comment:
on second look I think the trigger will have to be part of the Dockerfile 🤔  , but I'll change that  in a second PR if needed

Reply:
I have limited context, but my hunch is that you'll want to trigger this from the Dockerfile